client/src/components/MessageTextBox/MessageTextBox.jsx

```diff
@@ -0,0 +1,73 @@
+import React, { useState } from 'react';
+import { TextField, Button } from '@mui/material';
+import axios from 'axios'
+import { Redirect } from 'react-router-dom'
+
+const MessageTextBox = (props) => {
+  const [messageText, setMessageText] = useState('')
+  const [isMessage, setIsMessage] = useState(false)
+  const [isRedirect, setIsRedirect] = useState(false)
+
+  const handleChange = e => {
+    setMessageText(e.target.value)
+
+    if (e.target.value.length > 0) {
+      setIsMessage(true)
+    } else {
+      setIsMessage(false)
+    }
+  }
+
+  const sendMessage = () => {
+    const data = { message: messageText, selectedCustomers: props.selectedCustomers }
+    console.log(data)
+    axios.post('http://localhost:1337/api/message', data)
+      .then(res => {
+        console.log(res.statusText)
+        // Redirect to Message Log Page
+        setIsRedirect(true)
+      })
+      .catch(err => {
+        console.log('Error sending text message', err)
+      })
+  }
+
+
+  if (isRedirect) {
```

Comment:
You will be able to remove this block of code is you use the hook from react router